source/blog/2017-03-30-minishift-intro.html.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Introducing Minishift - Run OpenShift locally
+date: 2017-03-30 05:45 UTC
+author: budhram
+published: true
+comments: true
+tags: minishift, atomic-developer-bundle, openshift-local
+---
+
+It has been quite a few months since our [last blog post](http://www.projectatomic.io/blog/2016/08/vagrant-service-manager-1-3-0-release/) on [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) as we have been looking around the alternative of our [Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle). It would be a separate blog post to address this reason.
+
```

Comment:
we have been looking around the alternative of our [Atomic Developer Bundle (ADB)]> we have been looking around for an alternative to [Atomic Developer Bundle (ADB)].
Better would be> we have been looking around for alternative solutions to replace the [Atomic Developer Bundle (ADB)].
A separate blog post will address the reasons for this.